python-packages/sra_client/sra_client/__init__.py

```diff
@@ -2,6 +2,145 @@
 
 # flake8: noqa
 
+"""Demonstration of using the python sra_client.
+
+Configure and create an API client instance
+--------------------------------------------
+
+>>> from sra_client import ApiClient, Configuration
+>>> from sra_client.api import DefaultApi
+>>> config = Configuration()
+>>> config.host = "http://localhost:3000"
+>>> relayer_api = DefaultApi(ApiClient(config))
+
+Post Order
+-----------
+
+Post an order to an SRA-compliant Relayer.
+
+
+>>> import sra_client
+>>> example_signed_order = {
+...     "makerAddress": "0x5409ed021d9299bf6814279a6a1411a7e866a631",
+...     "takerAddress": "0x0000000000000000000000000000000000000000",
+...     "senderAddress": "0x0000000000000000000000000000000000000000",
+...     "exchangeAddress": "0x35dd2932454449b14cee11a94d3674a936d5d7b2",
+...     "feeRecipientAddress":
+...         "0x0000000000000000000000000000000000000000",
+...     "makerAssetData": "0xf47261b0000000000000000000000000"
+...         "d0a1e359811322d97991e03f863a0c30c2cf029c",
+...     "takerAssetData": "0xf47261b0000000000000000000000000"
+...         "2002d3812f58e35f0ea1ffbf80a75a38c32175fa",
+...     "salt": "2362734632784682376287462",
+...     "makerFee": "0",
+...     "takerFee": "0",
+...     "makerAssetAmount": "1000000000000000000",
+...     "takerAssetAmount": "500000000000000000000",
+...     "expirationTimeSeconds": "999999999999999999999",
+...     "signature": (
+...         "0x1cb085506ccef3d15061766808a6d5b5369a6dacc323101f704ab1b38d0166725"
```

Comment:
Use our signature utils to generate this?

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/338895ef102cb7508d6b41640d7f43d057743f59